plugins/arDominionB5Plugin/modules/repository/templates/_holdingsInstitution.php

```diff
@@ -1,38 +1,24 @@
-<section id="repo-holdings" class="list-menu"
-  data-total-pages="<?php echo $pager->getLastPage(); ?>"
-  data-url="<?php echo url_for(['module' => 'repository', 'action' => 'holdingsInstitution', 'id' => $resource->id]); ?>">
+<section class="card mb-4">
 
-  <div class="panel panel-gray">
-    <div class="panel-body">
+  <div class="p-4 pb-0">
```

Comment:
Maybe remove `mb-4` from the parent and `pb-0` from this element.

Reply:
Also, I think it may look better with `text-center` in this case inside the card. Just a note, we can check it later with the team to see if we want to do that and in which cases.

Reply:
> Maybe remove mb-4 from the parent and pb-0 from this element.

Like this?

![image](https://user-images.githubusercontent.com/606459/129525660-9b23b00b-9bf0-47ef-abe9-672391c8d347.png)